styles.css

```diff
@@ -197,6 +217,7 @@ footer > hr {
   margin-top: 12px;
   background: #ff6b00;
   border: none;
+  transition: all 0.7s;
 }
 
 form > .projectButton {
```

Comment:
- [ ] You did an amazing job replicating the Figma design. :+1: Let's work on a minor detail to match the design template. I think it would be better if you move the button to the left so it can get aligned with the other inputs above it. ✍️

**Your design**
![Screenshot from 2023-06-10 00-15-11](https://github.com/Yulianav09/Portfolio/assets/17243325/48f7cf9b-319f-4579-bca0-89e919e10142)

**Figma design**
![Screenshot from 2023-06-10 00-14-45](https://github.com/Yulianav09/Portfolio/assets/17243325/7dbe0cfb-49f8-446b-a5b5-108ff495bd73)